spec/models/article_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe Article, :type => :model do
+  context 'Create new article.' do
+    text = Faker::Lorem.words(100).join('')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.